examples/SampleApp/ios/Podfile.lock

```diff
@@ -860,7 +859,10 @@ SPEC CHECKSUMS:
   FlipperKit: cbdee19bdd4e7f05472a66ce290f1b729ba3cb86
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
   glog: 04b94705f318337d7ead9e6d17c019bd9b1f6b1b
+<<<<<<< HEAD
```

Comment:
![image](https://user-images.githubusercontent.com/11586388/222187511-10642743-dd86-4a11-bdd1-5d9b3a780170.png)

Reply:
Damn, let me fix it quickly.